liveproxy/server.py

```diff
@@ -596,6 +596,7 @@ def finish_request(self, request, client_address):
 
 class ThreadedHTTPServer(ThreadingMixIn, Server):
     '''Handle requests in a separate thread.'''
+    address_family = socket.AF_INET6
```

Comment:
breaks IPv4 `AF_INET`
stream won't open

Reply:
The underlying library here only binds one or the other then it seems. That's pretty lame. I'll see if I can make this work some other way. It should be that it'll bind both v4 and v6 by default